requirements.txt

```diff
@@ -1,4 +1,5 @@
 torch >= 1.9
 numpy
 Pillow
-regex
\ No newline at end of file
+regex
+tqdm
```

Comment:
Actually not this point, but we need to mention on README.md that we use tqdm.

Reply:
I will add a new commit to the PR adding `tqdm` among the dependencies listed in the README :)

Reply:
resolved.